03_Modules/Transactions/test/module/CostNotifier.test.ts

```diff
@@ -0,0 +1,156 @@
+import { faker } from '@faker-js/faker';
+import { expect, jest } from '@jest/globals';
+import { ITransactionEventRepository, Transaction } from '@citrineos/data';
+import { AbstractModule, CallAction } from '@citrineos/base';
+import { CostCalculator } from '../../src/module/CostCalculator';
+import { CostNotifier } from '../../src/module/CostNotifier';
+import { aTransaction } from '../providers/Transaction';
```

Comment:
This file is missing, preventing a build

Reply:
I made an adjustment, I think this may have just been accidentally renamed.. let's see if the pipeline succeeds, but it build locally for me 🙏